client/signup/steps/site-type/style.scss

```diff
@@ -0,0 +1,105 @@
+.site-type__wrapper {
+	position: relative;
+
+	strong {
+		color: rgb( 51, 69, 82 );
+		font-size: 1.2em;
+		display: block;
+	}
+
+	.form-label {
+		margin-bottom: 20px;
+	}
+
+	.form-radio {
+		margin-top: 15px;
+		margin-right: 15px;
+	}
+}
+
+.site-type__form-wrapper {
+	max-width: 500px;
+	margin: 0 auto;
+	filter: blur( 0 );
+	transform: translateZ( 0 ) translateX( 0 );
+	transition: 500ms ease-in-out opacity, 500ms ease-in-out filter, 500ms ease-in-out transform;
+}
+
+.site-type__wrapper-is-hidden {
+	pointer-events: none;
+	transform: translateZ( 0 ) translateX( 25% );
+	opacity: 0;
+}
+
+.site-type__segmented-control-wrapper {
+	display: flex;
+	justify-content: space-evenly;
+	align-items: center;
+}
+
+.site-type__segmented-control {
+	flex-grow: 1;
+}
+
+.site-type__segment-label {
+	color: $gray-darken-20;
+
+	&:hover {
+		cursor: pointer;
+	}
+}
+
+.site-type__min-label {
+	padding-right: 15px;
+}
+
+.site-type__max-label {
+	padding-left: 15px;
+}
+
+.site-type__submit-wrapper {
+	text-align: center;
+	margin-bottom: 0;
+	padding: 0 15px;
+}
+
+.site-type__submit-wrapper .button {
+	width: 180px;
+	@include breakpoint( '<480px' ) {
+		width: 100%;
+	}
+}
+
+.site-type__last-fieldset {
+	margin-bottom: 0;
+}
+
+h3 {
```

Comment:
this is affecting every h3 in calypso. we'll need to scope it to this particular component.